qiskit/utils/__init__.py

```diff
@@ -30,6 +30,13 @@
 .. autofunction:: wrap_method
 
 
+Parallel Routines
+-----------------
+A helper function for calling a custom function with python
+``ProcessPoolExecutor``. Tasks can be executed in parallel using this function.
```

Comment:
We have intersphinx set up to target the stdlib, so if you really want you can do
```suggestion
:class:`~concurrent.futures.ProcessPoolExecutor`. Tasks can be executed in parallel using this function.
```
not necessary, though

Reply:
Oh, that's a good idea. I'll do that